File: cpp/Problem136.cpp
/*
 * Problem: 136. Single Number
 * URL: https://leetcode.com/problems/single-number/
 * Difficulty: Easy
 * Tags: Bitwise Operation
 */

#include <vector>

using namespace std;

class Solution {
public:
	int singleNumber(vector<int>& nums) {
		int result = 0;
		for (int num : nums) {
			result ^= num;
		}
		return result;
	}
};

File: python/Problem136.py
# Problem: 136. Single Number
# URL: https://leetcode.com/problems/single-number/
# Difficulty: Easy
# Tags: Bitwise Operation

from typing import List

class Solution:
  def singleNumber(self, nums: List[int]) -> int:
    result: int = 0
    for num in nums:
      result ^= num
    return result